operations/host_spawn.go

```diff
@@ -1018,6 +1029,35 @@ func printHosts(hosts []*restModel.APIHost) {
 	}
 }
 
+func printHostsJSON(hosts []*restModel.APIHost) {
+	type hostResult struct {
+		Id               string `json:"id"`
+		Name             string `json:"name"`
+		Distro           string `json:"distro"`
+		Status           string `json:"status"`
+		HostName         string `json:"hostName"`
+		User             string `json:"user"`
+		AvailabilityZone string `json:"availabilityZone"`
```

Comment:
We use snake case by convention for json, so can these be host_name and availability_zone?